doc/operations/ledger_snapshot.rst

```diff
@@ -86,3 +86,15 @@ A low value for ``ledger.chunk_size`` means that smaller ledger files are genera
 Similarly, a low value for ``snapshots.tx_count`` means that snapshots are generated often and that join/recovery time will be short, at the cost of additional workload on the primary node for snapshot generation.
 
 .. tip:: Uncommitted ledger files (which are likely to contain committed transactions) can also be used on join/recovery, as long as they are copied to the node's ``ledger.directory`` directory.
+
+Invariants
+----------
+
+1. To facilitate audit and verification of the integrity of the ledger, individual ledger files always end on a signature transaction.
+
+2. For operator convenience, all committed ledger files (``.committed`` suffix) are the same on all up-to-date nodes.
```

Comment:
I think this could be slightly clearer/stronger. ie all committed transactions are present in exactly 1 `.committed` file (no duplicates), `.committed` files start and end at the same seqnos on every node (consistent chunking), and files with the same name on different nodes are _identical_ (no node-specific headers/contents in ledger). All caveated by "on the nodes where these files are present", or "all up-to-date nodes", or some other way of describing this.

Reply:
Good points. Wrt "all committed transactions are present in exactly 1 .committed file", I think this is bit confusing as 
since some committed transactions will be stored in non-committed chunks (explained at the bottom of this section (https://microsoft.github.io/CCF/main/operations/ledger_snapshot.html#file-layout). I've added the other points as sub-points in the `2.` invariant.